libkbfs/folder_block_manager.go

```diff
@@ -970,8 +974,8 @@ func (fbm *folderBlockManager) doReclamation(timer *time.Timer) (err error) {
 	fbm.log.CDebugf(ctx, "Starting quota reclamation process")
 	defer func() {
 		fbm.log.CDebugf(ctx, "Ending quota reclamation process: %v", err)
-		fbm.lastReclamationTimeLock.Lock()
-		defer fbm.lastReclamationTimeLock.Unlock()
+		fbm.lastQRLock.Lock()
+		defer fbm.lastQRLock.Unlock()
```

Comment:
I'm a bit worried that `fbm.lastReclamationTime` is updated non-atomically with the other fields, although I can't think of any concrete cases where this would lead to an error. Can you make it update atomically, anyway? Maybe have a `lastReclamationTime` var before the first defer and update `fbm.lastReclamationTime` in the defer if `lastReclamationTime` is non-nil